mscore/playpanel.ui

```diff
@@ -531,94 +575,83 @@
-          <property name="icon">
-           <iconset resource="musescore.qrc">
-            <normaloff>:/data/icons/media-playback-metronome.svg</normaloff>:/data/icons/media-playback-metronome.svg</iconset>
-          </property>
-         </widget>
-        </item>
-        <item>
-         <spacer name="horizontalSpacer_4">
-          <property name="orientation">
-           <enum>Qt::Horizontal</enum>
-          </property>
-          <property name="sizeHint" stdset="0">
-           <size>
-            <width>40</width>
-            <height>20</height>
-           </size>
-          </property>
-         </spacer>
-        </item>
-       </layout>
-      </item>
-      <item row="1" column="1">
-       <widget class="QLabel" name="tempoLabel">
+      <item row="2" column="0">
+       <widget class="QLabel" name="MetronomeTag">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
-        <property name="font">
-         <font/>
+        <property name="maximumSize">
+         <size>
+          <width>16777215</width>
+          <height>32</height>
+         </size>
         </property>
-        <property name="toolTip">
-         <string>Actual tempo in quarter notes per minute</string>
+        <property name="text">
+         <string>Metronome</string>
         </property>
-        <property name="frameShape">
-         <enum>QFrame::NoFrame</enum>
+        <property name="alignment">
+         <set>Qt::AlignCenter</set>
         </property>
-        <property name="lineWidth">
-         <number>2</number>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="Awl::VolSlider" name="mgainSlider" native="true">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
         </property>
-        <property name="midLineWidth">
-         <number>2</number>
+        <property name="minimumSize">
+         <size>
+          <width>0</width>
+          <height>64</height>
+         </size>
         </property>
-        <property name="text">
-         <string notr="true">120 BPM</string>
+        <property name="maximumSize">
+         <size>
+          <width>128</width>
+          <height>512</height>
+         </size>
         </property>
-        <property name="alignment">
-         <set>Qt::AlignCenter</set>
+        <property name="focusPolicy">
+         <enum>Qt::TabFocus</enum>
         </property>
-        <property name="textInteractionFlags">
-         <set>Qt::NoTextInteraction</set>
+        <property name="toolTip">
+         <string>Metronome volume</string>
+        </property>
+        <property name="accessibleName">
+         <string>Metronome Volume</string>
```

Comment:
better change to `Metronome volume`, same string as the tooTip, makes translation easier as it gets detected as a duplicate and the 'recycles' the translation, i.e. only one transltion for 2 strings. And capitalization isn't important at all for accessibility texts, i.e. for screen readers

Reply:
Thanks. Done.